core/templates/dev/head/pages/exploration_editor/statistics_tab/state_stats_modal_directive.html

```diff
@@ -1,5 +1,5 @@
 <div class="modal-header">
-  <h3>Statistics for "<[stateName]>"</h3>
+  <h3>Statistics for "<a ng-click="navigateToStateEditor()" ng-bind="stateName"></a>"</h3>
```

Comment:
Instead of using ng-bind I think you can do `<a ...><[stateName]></a>`.

Reply:
(assuming you have a $scope.stateName defined)

Reply:
Done.